.travis.yml

```diff
@@ -36,9 +38,9 @@ script:
 branches:
   only:
   - master
-  - /^(dawn)\/.*$/
-  - /^(runtime)\/.*$/
-  - /^(test)\/.*$/
+  - /^dawn\/.*$/
+  - /^runtime\/.*$/
+  - /^test\/.*$/
 deploy:
```

Comment:
There's a chance that I don't know how the deploy globs work (since I haven't used travis much), but I would like to know that you tested the release procedure after making this change.